lms/djangoapps/grades/new/course_grade.py

```diff
@@ -328,29 +347,42 @@ def update(self, course):
         course_structure = get_course_blocks(self.student, course.location)
         self._compute_and_update_grade(course, course_structure)
 
-    def _compute_and_update_grade(self, course, course_structure, read_only=False):
+    def get_persisted(self, course):
         """
-        Freshly computes and updates the grade for the student and course.
-
-        If read_only is True, doesn't save any updates to the grades.
+        Returns the saved grade for the given course and student,
+        irrespective of whether the saved grade is up-to-date.
         """
-        course_grade = CourseGrade(self.student, course, course_structure)
-        course_grade.compute_and_update(read_only)
-        return course_grade
+        if not PersistentGradesEnabledFlag.feature_enabled(course.id):
+            return None
+
+        return CourseGrade.get_persisted_grade(self.student, course)
 
-    def _get_saved_grade(self, course, course_structure):  # pylint: disable=unused-argument
+    def _get_saved_grade(self, course, course_structure=None):
```

Comment:
Can we revert this change since it's no longer necessary to have `course_structure` be an optional argument?  Also, the `unused-argument` may no longer be needed. (?)